compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/PipelineTranslationContext.java

```diff
@@ -275,7 +275,7 @@ DAGBuilder getBuilder() {
     if (srcTransform instanceof FlattenTransform) {
       return CommunicationPatternProperty.Value.ONE_TO_ONE;
     }
-    if (dstTransform instanceof GBKTransform
+    if ((dstTransform instanceof GBKTransform && !(dstTransform instanceof CombineFnWindowedTransform))
```

Comment:
How about adding a variable like `private final paritalCombining = false` and getter method like `isPartialCombiner()`? in`GBKTransform`? so here we can check whether the GBKTransform is partialCombiner or not.

Reply:
I think that's better than the previous implementation. I'll implement it as you described.